modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -147,11 +147,11 @@ ifdef::baremetal,baremetal-restricted[]
 If you are running a three-node cluster, skip the following step to allow the masters to be schedulable.
 ====
 endif::baremetal,baremetal-restricted[]
-. Modify the `<installation_directory>/manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file to prevent pods from being scheduled on the control plane machines:
+. Check that the `mastersSchedulable` parameter in the `<installation_directory>/manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file is set to `false`. This prevents pods from being scheduled on the control plane machines:
```

Comment:
Due to the removal of the step discussed here, the `mastersSchedulable` parameter in the `<installation_directory>/manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file should already by set to `false`. But, just in case it is not, I reworded this step instead of taking it out.